src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int SUPPLY_OR_BUY = 0;
+    private static final int SUPPLY_OR_BUY_VALUE = 1;
+    private static final String CSV_SEPARATOR = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] infoFromFile = readFile(fromFileName);
+        String[] splitedLine;
+        int supply = 0;
+        int buyAmount = 0;
+        for (String splitTextToLine : infoFromFile) {
+            splitedLine = splitTextToLine.split(CSV_SEPARATOR);
+            if ("supply".equals(splitedLine[SUPPLY_OR_BUY])) {
+                supply += Integer.parseInt(splitedLine[SUPPLY_OR_BUY_VALUE]);
+            } else {
+                buyAmount += Integer.parseInt(splitedLine[SUPPLY_OR_BUY_VALUE]);
+            }
+        }
+        int remain = supply - buyAmount;
+        StringBuilder result = new StringBuilder();
+        String statistic = result.append("supply,").append(supply).append(System.lineSeparator())
```

Comment:
Коли створиш константи результат можна переписати так
```suggestion
        String statistic = result.append(SUPPLY).append(CSV_SEPARATOR).append(supply).append(System.lineSeparator())
```